electron-app/package.json

```diff
@@ -64,9 +66,11 @@
     "megalodon": "^6.1.0",
     "nanoid": "^2.1.8",
     "nedb": "^1.8.0",
+    "node-fetch": "^2.6.12",
```

Comment:
Any particular reason this is required? The app has requestjs (as unwieldy as it may be)

Reply:
Bluesky lib uses fetch internally